src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -405,8 +405,8 @@
   <data name="MultiLinePasteDialog.CloseButtonText" xml:space="preserve">
     <value>Cancel</value>
   </data>
-  <data name="MultiLinePasteDialog.Content" xml:space="preserve">
-    <value>You are about to paste text that contains multiple lines. If you paste this text into your shell, it may result in the unexpected execution of commands. Do you wish to continue?</value>
+  <data name="MultiLineWarningText.Text" xml:space="preserve">

```

Comment:
i wish we didn't have to change the key for this. I know _why_ we have to, but this will trigger a re-localization for a string that is almost exactly the same.